addon/mixins/legend.js

```diff
@@ -91,32 +98,44 @@ export default Ember.Mixin.create({
     };
   }),
 
-  // Place each legend item, breaking across rows. Center them if there is one
-  // row
-  // Ideally legend items would be centered to the very middle of the graph,
-  // this is made difficult by the fact that we want the icons to line up in
-  // nice columns and that the labels are variable length
-  legendItemAttrs: Ember.computed('legendItemWidth', 'legendItemHeight', 'numLegendItemsPerRow',
-     'legendItems.length', 'numLegendRows', 'averageLegendLabelWidth', function() {
-
+  // Place each legend item such that the legend rows appear centered to the graph.
+  // Spacing between legend items must be constant and equal to 2*legendLabelPadding = 20px.
+  // This breaks the old behavior where legend icons are placed in nice columns but caused
```

Comment:
Don't talk about old behavior. If it's not in the code anymore, it's not relevant.